astroquery/esa/hubble/core.py

```diff
@@ -239,7 +322,7 @@ def _get_product_filename(self, product_type, filename):
         else:
             return f"{filename}.zip"
 
-    def get_artifact(self, artifact_id, *, filename=None, verbose=False):
+    def get_artifact(self, artifact_id, *, filename=None, folder=os.getcwd(), verbose=False):
```

Comment:
same here (and everywhere else below, too, maybe use `None` as default that defaults to current dir

Reply:
Removed and replaced with None.